browser/main/lib/dataApi/resolveStorageData.js

```diff
@@ -32,12 +32,11 @@ function resolveStorageData (storageCache) {
   const version = parseInt(storage.version, 10)
   if (version >= 1) {
     if (version > 1) {
-      console.log('The repository version is newer than one of current app.')
+      return
```

Comment:
return will not create a promise anymore! that changes the baviour of this method!